src/components/modals/modal.module.scss

```diff
@@ -1,7 +1,18 @@
+@import 'styles/sfds/sfdsDependencies';
+
 .buttonGroupWithDelete {
   // Right-align last child
   > :last-child {
     flex-grow: 1;
     text-align: right;
   }
 }
+
+.customLinkModal {
```

Comment:
Thought: The cancel link is pretty hard to read for me in dark mode. I wonder if it needs a different color or if that's taken care of else where. I suppose it's another thing to add to the list for design to review. That said I don't see why it has to block this PR.

Reply:
Agreed. I didn't make any changes to it because I believe it will be resolved [here](https://github.com/USSF-ORBIT/ussf-portal-client/pull/770/files#diff-8ca7337033ae68929dee9564b1192523179972697ad582c609e14de8fe761df7). I still think we should bring it up to design though, because I'm still not sure what the hover state should look like for a handful of things.